cri/v1alpha2/cri_wrapper.go

```diff
@@ -56,6 +56,20 @@ func (c *CriWrapper) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 	return c.CriManager.RunPodSandbox(ctx, r)
 }
 
+// StartPodSandbox start a sandbox pod which was forced to stop by external factors.
+// Network plugin returns same IPs when input same pod names and namespaces
```

Comment:
Hmm... the comment seems wired to me 😂 

Maybe the comment like this is more natural:

StartPodSandbox restart a sandbox pod which was stopped by accident and we should reconfigure it with network plugin which will make sure it reacquire its original network configuration, like IP address.

@Starnop @fuweid WDYT?

If it make any sense, please update other docs as well :)

Reply:
Has been updated. ^_^